src/test/resources/pacts/publicapi-direct-debit-connector-search-mandates-endpoint.json

```diff
@@ -0,0 +1,220 @@
+{
+  "consumer": {
+    "name": "publicapi"
+  },
+  "provider": {
+    "name": "connector"
+  },
+  "interactions": [
+    {
+      "description": "search mandates by parameters",
+      "providerStates": [
+        {
+          "name": "a gateway account with an external ids and a confirmed mandate with an email exists",
+          "params": {
+            "gateway_account_id": "2po9ycynwq8yxdgg2qwq9e9qpyrtre",
+            "mandate_id": "jkdjsvd8f78ffkwfek2a",
+            "unique_identifier": "AAAAA",
+            "charge_id": "charge97837509646393e3A",
+            "email": "rory@example.com"
+          }
+        }
+      ],
+      "request": {
+        "method": "GET",
+        "path": "/v1/api/accounts/123456/mandates",
+        "query": {
+          "email": [
+            "rory@example.com"
+          ]
+        }
+      },
+      "response": {
+        "status": 200,
+        "headers": {
+          "Content-Type": "application/json"
+        },
+        "body": {
+          "total": 1,
+          "count": 1,
+          "page": 1,
+          "results": [
+            {
+              "mandate_id": "obck2jlpu4erf3j2itfkp215ld",
+              "provider_id": "GOVERNMENTDI-ZY79J",
+              "reference": "aASasa",
+              "return_url": "https://www.example.com",
+              "payment_provider": "paymentProviderId",
+              "bank_statement_reference": "AAAA1111",
+              "created_date": "1995-10-27T10:37:19.753Z",
+              "payer": {
+                "name": "Mr Test",
+                "email": "rory@example.com"
+              },
+              "state": {
+                "status": "a status",
+                "details": "cause_details_from_gocardless"
+              },
+              "_links": {
```

Comment:
I think the links for each result should just be `links` and its an array...quick test with pay local on latest DDConnector master...
```
"links": [
                {
                    "rel": "self",
                    "method": "GET",
                    "href": "http://localhost:10100/v1/api/accounts/DIRECT_DEBIT:lmcd4n087p5moa8j4kplrv5djo/mandates/g68qrb0ucvpfd3jtochq1rt2g5"
                },
                {
                    "rel": "next_url",
                    "method": "GET",
                    "href": "http://localhost:10200/secure/01dd76e0-c591-451b-bff5-2afef3d5bdfc"
                },
                {
                    "rel": "next_url_post",
                    "method": "POST",
                    "href": "http://localhost:10200/secure",
                    "type": "application/x-www-form-urlencoded",
                    "params": {
                        "chargeTokenId": "01dd76e0-c591-451b-bff5-2afef3d5bdfc"
                    }
                }
            ],
```